src/components/movable-table/MovableTable.tsx

```diff
@@ -76,70 +36,54 @@ export const MovableTable = ({ columns, data, setData, entityRows }: MovableTabl
         values={data}
         onChange={({ oldIndex, newIndex }) => onChange(oldIndex, newIndex)}
         renderList={({ children, props, isDragged }) => (
-          <table
-            style={{
-              ...tableStyles,
-              cursor: isDragged ? 'grabbing' : undefined,
-              width: '100%',
-            }}
-          >
+          <StyledTable $isDragged={isDragged}>
             <thead>
               <tr>
                 {columns.map((column, index) => (
-                  <th
+                  <StyledTableHeadCell
                     key={`th-${index}`}
-                    style={{
-                      ...tableHeadCellStyles,
-                      width: column.width ?? 'auto',
-                      textAlign: column?.type === BasicTableColumnType.Financial ? 'right' : 'left',
-                      ...getSidePadding(index, columns.length),
-                    }}
+                    $width={column.width ?? 'auto'}
+                    $textAlign={column?.type === BasicTableColumnType.Financial ? 'right' : 'left'}
+                    $index={index}
+                    $numberOfColumns={columns.length}
+                    $tableHeadCellStyles={tableHeadCellStyles}
+                    $getSidePadding={getSidePadding}
                   >
                     {column.label}
-                  </th>
+                  </StyledTableHeadCell>
                 ))}
               </tr>
             </thead>
             <tbody {...props}>{children}</tbody>
-          </table>
+          </StyledTable>
         )}
         renderItem={({ value, props, isDragged, isSelected }) => {
           const _widths = isDragged ? widths : [];
           const row = (
-            <tr
+            <StyledTableBodyRow
               {...props}
-              style={{
-                ...props.style,
-                cursor: isDragged ? 'grabbing' : undefined,
-                backgroundColor: isDragged || isSelected ? light6 : light4,
-                ...tableBodyRowStyles,
-                ...borderBottom(props.key === entityRows.length - 1 ? undefined : border300),
-              }}
+              $isDragged={isDragged}
+              $isSelected={isSelected}
+              $isBorderBottom={props.key === entityRows.length - 1}
             >
               {columns.map((column, index) => (
```

Comment:
Passing the getSidePadding does not seem to work. The 24 padding is not added anymore at the side. I think we can simplify it at make it work again by doing the following. ( also for the tablehead above ) 

```suggestion
              {columns.map((column, index) => { 
                   const sidePadding = getSidePadding(index, columns?.length);
                   return <StyledTableBodyCell
                    {...sidePadding}
                  
```